test/it/uk/gov/hmrc/apinotificationpull/acceptance/RetrieveAndDeleteNotificationSpec.scala

```diff
@@ -26,9 +26,9 @@ import play.api.test.Helpers._
 
 class RetrieveAndDeleteNotificationSpec extends FeatureSpec with GivenWhenThen with Matchers with GuiceOneAppPerTest {
 
-  feature("Retrieve a single message from the Pull Service (DELETE)") {
+  feature("Retrieve(DELETE) a single message from the API Notification Pull service") {
     info("As a 3rd Party")
-    info("I want to successfully receive any notifications waiting for me")
+    info("I want to successfully receive all notifications waiting for me")
```

Comment:
This test covers retrieving a single notification. Perhaps this should read something like `info("I want to successfully receive a notification waiting for me")`

Reply:
fair enough